Formula/libmpdec.rb

```diff
@@ -0,0 +1,49 @@
+class Libmpdec < Formula
```

Comment:
```suggestion
class Mpdecimal < Formula
```

Reply:
That means you'll also need to change the filename